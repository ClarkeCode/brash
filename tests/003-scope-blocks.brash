var name = "Greg"
{
	var num = 2.456
	print name
	print num
}
