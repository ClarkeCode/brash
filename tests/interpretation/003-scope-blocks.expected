Greg
2.456000
